app/templates/document_download_template.html

```diff
@@ -11,7 +13,7 @@
 
 {% block head %}
   <link rel="stylesheet" media="screen" href="{{ asset_url('stylesheets/main.css') }}" />
-  <style>
+  <style nonce="{{ request.csp_nonce }}"> 
```

Comment:
as you're already creating a variable called `cspNonce`, are you able to use it here, as it `<style nonce="{{ cspNonce }}">` ?